Lib/test/test_weakref.py

```diff
@@ -1919,6 +1919,7 @@ def test_threaded_weak_valued_pop(self):
             for i in range(100000):
                 if i%1000==0:
                     print("\rLoop:"+str(i)+"/100000     ", end="")
+                    sys.stdout.flush()
```

Comment:
Did you add these lines to the CPython test? If so, please add a `# TODO: RUSTPYTHON` annotation that explains your reasoning.

Note to self (and others interested): `Lib/test/support/threading_helper.py` and this file need updates to CPython 3.11 (there's a new `@threading_helper.requires_working_threading` annotation that tests for WASI and emscripten).

Reply:
Added explanatory comment for using flush here

> Did you add these lines to the CPython test? If so, please add a `# TODO: RUSTPYTHON` annotation that explains your reasoning.
> 
> Note to self (and others interested): `Lib/test/support/threading_helper.py` and this file need updates to CPython 3.11 (there's a new `@threading_helper.requires_working_threading` annotation that tests for WASI and emscripten).